README.md

```diff
@@ -71,10 +71,10 @@ At this point, you should be ready to start your app in development mode.
 3. As long as you leave `SUPPRESS_SELF_INVITE=` blank in your `.env` you should be able to invite yourself from the homepage.
    - If you DO set that variable, then spoke will be invite-only and you'll need to generate an invite. Run:
      ```
-     echo "INSERT INTO invite (hash,is_valid) VALUES ('abc', 1);" |sqlite3 mydb.sqlite
+     echo "INSERT INTO invite (hash,is_valid) VALUES ('E4502B28-301E-4E63-9A97-ACA14E8160C8', 1);" |sqlite3 mydb.sqlite
      # Note: When doing this with PostgreSQL, you would replace the `1` with `true`
```

Comment:
```suggestion
     # Notes: When doing this with PostgreSQL, you would replace the `1` with `true`, in this example the invite is a hashed version of the string 'abc'
```

Reply:
This isn't a hash of 'abc' -- it's just a random uuid.

Reply:
Ah yep, after the conversation we had in the other comment I realized I was mixed up. I'll resolve!